app/views/quick_search/search/index.html.erb

```diff
@@ -35,13 +35,22 @@
       </div>
     </div>
   <% end %>
-  <div class="col library-website module">
```

Comment:
@mejackreed can we retain the partial for the old searcher? (i.e. add an ENABLED_SEARCHERS block for library_website). Let's assume that we'll never have both services enabled. It's going to take a bit of time to compare results between the two until we can fully remove it.

Reply:
Yes, I think it is still right below Line 47? Is that what you mean?

Reply:
I may have gotten confused on a rebase